action.yml

```diff
@@ -25,6 +25,9 @@ inputs:
     description: 'Tag of the built image'
     required: false
     default: ''
+  archs:
+    description: 'architecture to build on'
+    required: true
```

Comment:
```suggestion
    required: false
    default: 'x86_64'
```

Reply:
Yes i will make it optional & set default as 'x86_64'